ui/components/ui/nft-info/nft-info.js

```diff
@@ -24,36 +22,25 @@ export default function NftInfo({ assetName, tokenAddress, tokenId }) {
     >
       <Box display={DISPLAY.FLEX} className="nft-info__content">
         <Box margin={4}>
-          <Identicon address={tokenAddress} diameter={24} />
+          <NftCollectionImage
+            assetName={assetName}
+            tokenAddress={tokenAddress}
+          />
         </Box>
         <Box>
-          <Typography
-            fontWeight={FONT_WEIGHT.BOLD}
-            variant={TypographyVariant.H6}
-            marginTop={4}
-          >
-            {assetName}
-          </Typography>
-          <Typography
-            variant={TypographyVariant.H7}
+          <Text variant={TextVariant.bodyMdBold} marginTop={4} as="h6">
+            {assetName ?? t('unknownCollection')}
+          </Text>
+          <Text
+            variant={TextVariant.bodySm}
             marginBottom={4}
             color={TextColor.textAlternative}
+            as="h6"
           >
-            {t('tokenId')} #{tokenId}
-          </Typography>
+            {`${t('tokenId')} #${tokenId}`}
+          </Text>
         </Box>
       </Box>
-      <Box marginTop={4} marginRight={4}>
-        <Button className="nft-info__button" type="link">
-          <Typography
-            variant={TypographyVariant.H6}
-            marginTop={0}
-            color={TextColor.primaryDefault}
-          >
-            {t('view')}
-          </Typography>
-        </Button>
-      </Box>
     </Box>
   );
 }
```

Comment:
I think you can save a wrapping div here

```jsx
  return (
    <Box
      className="nft-info"
      backgroundColor={BackgroundColor.backgroundAlternative}
      padding={4}
      display={DISPLAY.FLEX}
    >
      <NftCollectionImage assetName={assetName} tokenAddress={tokenAddress} />
      <Box marginLeft={4}>
        <Text variant={TextVariant.bodyMdBold} as="h6">
          {assetName ?? t('unknownCollection')}
        </Text>
        <Text
          variant={TextVariant.bodySm}
          color={TextColor.textAlternative}
          as="h6"
        >
          {`${t('tokenId')} #${tokenId}`}
        </Text>
      </Box>
    </Box>
  );
```

<img width="1440" alt="Screenshot 2023-03-28 at 7 35 14 PM" src="https://user-images.githubusercontent.com/8112138/228411915-ba47ed1f-4af8-492c-b2b7-348ea6294b56.png">

<img width="1439" alt="Screenshot 2023-03-28 at 7 34 11 PM" src="https://user-images.githubusercontent.com/8112138/228411909-bd0c162c-aeba-451c-af7a-c45a8c9693c1.png">